src/gui/plugins/view_angle/ViewAngle.qml

```diff
@@ -391,6 +391,41 @@ ColumnLayout {
     }
   }
 
+  // Set camera's FOV
+  Text {
```

Comment:
We need to increase the minimum height of `ViewAngle`, otherwise the new fields don't fit when the plugin is compressed:

![scroll_angle](https://user-images.githubusercontent.com/5751272/169152984-4f8ea454-6d88-464c-bc03-2cfcb5c65e15.gif)